src/input-mask.test.ts

```diff
@@ -56,6 +56,19 @@ QUnit.test('Should mask telephone input and strip letters from entry', assert =>
   assert.equal(inputElement.value, '(123) 4  -    ');
 });
 
+QUnit.test('Should mask telephone input on blur', assert => {
```

Comment:
@GarrettB71 were you able to get this test to pass? The utilities have been updated to not use underscores in the phone number mask.

Reply:
Fixed!